six7/figma-tokens
# Swap styles

Allows users to swap styles to the token's siblings - all without having tokens applied.

To test:
- Have a file with Themes setup and connected styles (Light / Dark theme) with `Prefix styles with theme name` so that there's both Light and Dark styles in the file
- Have Styles of Light applied on layers (styles, not tokens!)
- Change theme to Dark while having these layers selected
- Notice how we're swapping the applied styles to the styles of the Dark theme
